replacement_files/config_structure.cpp

```diff
@@ -2277,6 +2277,18 @@ void CConfig::SetConfig_Options(unsigned short val_iZone, unsigned short val_nZo
   /* DESCRIPTION:  preCICE configuration file name */
   addStringOption("PRECICE_CONFIG_FILENAME", preciceConfigFileName, string("precice.xml"));
 
+  /* DESCRIPTION:  Name of the participant in the preCICE configuration file */
+  addStringOption("PRECICE_PARTICIPANT_NAME", preciceParticipantName, string("SU2"));
+
+  /* DESCRIPTION:  Name of the mesh in the preCICE configuration file */
+  addStringOption("PRECICE_MESH_NAME", preciceMeshName, string("SU2-Mesh"));
+
+  /* DESCRIPTION:  Name of the read data in the preCICE configuration file */
+  addStringOption("PRECICE_READ_DATA_NAME", preciceReadDataName, string("Displacements"));
```

Comment:
I assume that the last argument is the default value. I guess that using the old hard-coded values here (e.g. `Displacements0`) would make this change backwards-compatible. But this is only an observation, I don't see a real benefit in doing so.

Reply:
It would only work in case one has only a single interface. If this is not the case, one would have `Mesh01`... which might be a complete disaster,